pkg/proxy/iptables/proxier_test.go

```diff
@@ -4651,59 +4595,24 @@ func Test_EndpointSliceWithTerminatingEndpoints(t *testing.T) {
 					},
 				},
 			},
-			expectedIPTables: `
```

Comment:
I don't think we need this full iptables diffs everywhere, but they are remarkably helpful when understanding the full diff a change results in. Maybe it's worth leaving a couple more in place?

Reply:
Sure, if people want.